src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -74,7 +74,7 @@ class DepositServlet(app: EasyDepositApiApp)
         _ = logger.info(s"retrieve deposits for user '$userId'")
         deposits <- app.getDeposits(userId)
       } yield Ok(body = toJson(deposits), headers = Map(contentTypeJson))
-    }.getOrRecoverWithActionResult
+      }.getOrRecoverWithActionResult
```

Comment:
there is a lot of unintended formatting in this file. I know this needs to be fixed some time soon, but let's keep the diff to a minimum and restore these formatting changes.

Reply:
There was no unit test to test invalid UUID. Added now.